packages/tldraw/src/lib/ui/constants.ts

```diff
@@ -15,3 +15,5 @@ export enum PORTRAIT_BREAKPOINT {
 }
 
 export const ADJACENT_SHAPE_MARGIN = 20
+
+export const SHAPES_WHICH_ARROWS_CANNOT_BIND_TO = ['arrow', 'group']
```

Comment:
i'm wondering whether we should add `line` to this. It's fundamentally broken at the moment since the normalized anchor doesn't make any sense when the line can change within its own bounds. @steveruizok how do you feel about arrows being unable to bind to lines?

![Kapture 2024-05-20 at 16 20 07](https://github.com/tldraw/tldraw/assets/1242537/e388c8d1-ce80-4214-b8dd-491d64d7cef1)